packages/storefront/src/views/Page.vue

```diff
@@ -110,4 +151,49 @@
   p-tabs-bar {
     margin-bottom: $pds-spacing-fluid-large;
   }
+
+  .skeleton {
+    height: $pds-font-line-height;
+
+    &--h1 {
+      max-width: 200px;
```

Comment:
Not sure if `px` is best choice here, maybe `rem` would be better. How does it look in scale mode?

Reply:
Looks better with px because otherwise it is too wide.
<img width="770" alt="Bildschirmfoto 2023-10-11 um 13 33 02" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/a0852b31-31e3-4c26-92fc-12a8d663069c">

<img width="768" alt="Bildschirmfoto 2023-10-11 um 13 32 47" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/1c3e3c9f-816e-41ce-9703-d438175f2d48">